smartmin/csv_imports/models.py

```diff
@@ -19,6 +19,14 @@ def generate_file_path(instance, filename):
 
 
 class ImportTask(SmartModel):
+    PENDING = 'PENDING'
+    STARTED = 'STARTED'
+    RUNNING = 'RUNNING'
+    SUCCESS = 'SUCCESS'
+    FAILURE = 'FAILURE'
+
+    READY_STATES = [SUCCESS, FAILURE]
```

Comment:
Maybe `FINAL_STATES` here? Ready seems a bit ambiguous. Also why not use a single character here?  Maybe change `SUCCESS` to `COMPLETE` so there isn't overlap with `S`